torch/_functorch/_aot_autograd/runtime_wrappers.py

```diff
@@ -74,9 +74,23 @@ def create_runtime_wrapper(
     if not hasattr(compiled_fn, "_boxed_call"):
         compiled_fn = make_boxed_func(compiled_fn)
 
-    def runtime_wrapper(*args):
+    def runtime_wrapper(args):
```

Comment:
fyi @tfiala some other (conflicting) stuff going on around AOTAutograd runtime wrapper :)

The problem is that python is holding onto tensor references of the inputs for the lifetime of the runtime wrapper, which prevents inductor from clearing their memory part-way through execution. Horace has a great writeup on it here: https://github.com/pytorch/pytorch/pull/83137#issuecomment-1211320670 (and a new flavor of the problem is showing up in compiled autograd, which this PR addresses)